MdeModulePkg/Library/UefiHiiLib/HiiLib.c

```diff
@@ -1936,6 +1961,8 @@ GetBlockDataInfo (
   IFR_BLOCK_DATA  *BlockArray;
   UINT8           *DataBuffer;
 
+  UINT16  Sum1, Sum2, Sum3; // MU_CHANGE - CodeQL change

```

Comment:
Our style would normally place these on separate lines. I'm generally fine with things that pass uncrustify though. So feel free to resolve this if you don't want to change it.

Reply:
I'll leave it as is to keep line diffs consolidated